src/pages/home/platform-carousel/_details.tsx

```diff
@@ -76,20 +84,22 @@ const Details = ({ slide, platform_details }: DetailsProps) => {
                 />
             </Flex>
             <Flex>
-                {selected_platform?.download_links?.map((link, index) => {
-                    return (
-                        <DownloadLink
-                            key={index}
-                            external="true"
-                            type={link?.link_type}
-                            to={link?.url}
-                            target="_blank"
-                            rel="noopener noreferrer"
-                        >
-                            <img src={getOSIcon(link.type)} loading="lazy" />
-                        </DownloadLink>
-                    )
-                })}
+                <StyledFlex>
+                    {selected_platform?.download_links.is_desktop?.map((link, index) => {
+                        return (
+                            <DownloadLink
+                                key={index}
```

Comment:
Please don't use the map index as key, it's not efficient. please check [this out](https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9)